images/kube-base/v1.15.12/Dockerfile

```diff
@@ -17,6 +17,13 @@ COPY --from=base /usr/local/bin/clean-install /usr/local/bin/clean-install
 RUN apt-get update \
     && apt-get dist-upgrade -y
 
+# Fix CVEs
+RUN apt-get install -y --no-install-recommends \
```

Comment:
i think the clean-install script on line 14 is for installing packages

Reply:
The package has been fixed in the base image so I removed this